### PR TITLE
Update template with open source template 2.0 changes

### PR DESCRIPTION
**Pull request by KCeh**
## Summary

Updates to sync template with [Open-source 2.0 template release](https://github.com/infinum/open-source-project-template/releases/tag/2.0.0)

## Changes

Update markdown files with additional info

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [x] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -28,3 +37,5 @@ The repository files can be changed if the project requires it. If there is a pr
 - [ ] Update the `.gitignore` file
 - [ ] Update the pull request template
 - [ ] Update issue templates
+
+[^1]: Define how changes will be documented; e.g. `CHANGELOG.md` file, Releases section in GitHub, usage of additional tools, etc.
```
Comment (JonatanPlesko) [inline SETUP.md]: Hmm, could we suggest (strongly advise) to use the releases section in GitHub? It's a common standard and the simplest way of doing it, I think we only added this to the general template since some teams used some additional tools for this.

File: SETUP.md [hunk 2]
```diff
@@ -15,7 +15,16 @@ Before you start using this template, complete the following steps:
   - [ ] Add the project name
   - [ ] Add the missing information based on the comments in the file
 - [ ] Add code owners to the `.github/CODEOWNERS` file if applicable
+- [ ] Define a changelog strategy[^1]
 - [ ] Delete the `SETUP.md` file
+
+### Project information
+
+If your project needs to provide additional information like backward compatibility, system status or similar, include it to the `README.md` file of additional file in a clear and informative format (e.g. compatibility matrix, table, timeline, etc.). Also, if there is a need for additional information for contributors, additional security measures or specific license, update and adjust related files.
```
Comment (JonatanPlesko) [inline SETUP.md]: What is "system status" referring to?

**Review by JonatanPlesko — APPROVED** [review]

File: SETUP.md (hunk 1 above)
Comment (KCeh) [reply SETUP.md]: I agree, but I think we also have Changelog.md for most of our libs
But it is ok that we remove this

File: SETUP.md (hunk 2 above)
Comment (KCeh) [reply SETUP.md]: Is the system running?
I guess this makes more senes for backend projects

File: SETUP.md (hunk 1 above)
Comment (JonatanPlesko) [reply SETUP.md]: I think we have both, either way it makes sense to suggest something that we consider to be the usual standard

File: SETUP.md (hunk 2 above)
Comment (JonatanPlesko) [reply SETUP.md]: I would also remove this reference then, it's just confusing within the context of Android
